index.php

```diff
@@ -1,72 +1,121 @@
 <?php
 
-if (empty($_SERVER['HTTPS']) || $_SERVER['HTTPS'] === "off") {
-    $location = 'https://' . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'];
-    header('HTTP/1.1 301 Moved Permanently');
-    header('Location: ' . $location);
-    exit;
-}
+//TODO:: may want to keep commented out for development purposes, probably want to uncomment the code for production
```

Comment:
The code should *always* be in production state.  If you need code for development, please wrap it with a config variable like $disable_https_redirect OR some other method (I think Apache exposes whether or not it has https enabled on the server etc).

Reply:
Just to add more color to my previous comment:

Right now, someone could install the code from the main branch and get a production quality product -- allbeit with the insane php 5.6 requirement.

I'd like to keep the "this is not beta software" vibe as much as possible.  

Exception: Once we finish with changes, we'll merge this in and it will in theory be production but will require testing to double check.  Thats an acceptable compromise IMHO.  

But we should't check in code that takes it from production-ready to software-in-development status, if that makes sense.

Reply:
Again, oversight on my part as I was trying to get this out (of my hair). Will uncomment the code but can't address if the code will run. As it ran in the past *and* it does not appear to be tied to a particular PHP version, should be alright.